src/skin/firstRun.js

```diff
@@ -1,57 +0,0 @@
-(function($) {
-
-$(window).on("load", function () {
-  $('.jcarousel').jcarousel();
-
-  $('.jcarousel-control-prev')
-    .on('jcarouselcontrol:active', function() {
-      $(this).removeClass('inactive');
-    })
-    .on('jcarouselcontrol:inactive', function() {
-      $(this).addClass('inactive');
-    })
-    .jcarouselControl({
-      target: '-=1'
-    });
-
-  $('.jcarousel-control-next')
-    .on('jcarouselcontrol:active', function() {
-      $(this).removeClass('inactive');
-    })
-    .on('jcarouselcontrol:inactive', function() {
-      $(this).addClass('inactive');
-    })
-    .jcarouselControl({
-      target: '+=1'
-    });
-
-  $('.jcarousel-pagination')
-    .on('jcarouselpagination:active', 'a', function() {
-      $(this).addClass('active');
-    })
-    .on('jcarouselpagination:inactive', 'a', function() {
-      $(this).removeClass('active');
-    })
-    .jcarouselPagination();
-
-  $(document).on('keyup', function(e) {
-    var key = e.which || e.keyChar || e.keyCode;
-    if (key == 37) { // Left arrow
-      $('.jcarousel').jcarousel('scroll', '-=1');
-    } else if (key == 39) { // Right arrow
-      $('.jcarousel').jcarousel('scroll', '+=1');
-      setSeenComic();
```

Comment:
We should make sure to do the equivalent of `setSeenComic()` once we decide the user sufficiently interacted with the walkthrough, to avoid needlessly showing the walkthrough reminder in the popup.

Reply:
Not sure what this is or how it displays, will ask @squiggy .
Also @ghostwords  should I commit changes here or in the fork I created originally? Or does it not matter?

Reply:
Either one works. You avoid a merge if you commit to this PR directly.

Reply:
Regarding `setSeenComic()`: When you first install Privacy Badger, the walkthrough pops up. If you close it without interacting with the page, you will see a little reminder message next time you open Privacy Badger's popup from the browser toolbar. You shouldn't get that reminder if you did interact with the walkthrough though.

Reply:
I've put this in pb.js, based on if the user scrolls down the page.

Reply:
Could we rename `pb.js` to `firstRun.js`, to match what we had previously? Same request for the CSS file.

Reply:
Yes.